oauth2client/client.py

```diff
@@ -469,6 +467,11 @@ def clean_headers(headers):
     return clean
 
 
+def _parse_expiry(expiry):
+    if expiry and isinstance(expiry, datetime.datetime):
+        return expiry.strftime(EXPIRY_FORMAT)
```

Comment:
It's arbitrary in that I've never heard it recommended before. Maybe I meant subjective?

It seems to only protect Python beginners from misunderstanding, since `None` is the known return value.